src/mvPyUtils.cpp

```diff
@@ -2396,6 +2396,12 @@ GenerateCoreFile(std::ofstream& stream)
                 stream << "\n\t\twarnings.warn('" << args.name << " keyword renamed to " << args.new_name << "', DeprecationWarning, 2)";
                 stream << "\n\t\t" << args.new_name << "=kwargs['" << args.name << "']";
             }
+
+            else if (args.arg_type == mvArgType::DEPRECATED_KEYWORD_ARG)
+            {
+                stream << "\n\n\tif '" << args.name << "' in kwargs.keys():";
+                stream << "\n\t\twarnings.warn('" << args.name << " keyword deprecated." << args.new_name << "', DeprecationWarning, 2)";
```

Comment:
Let's add a space after "deprecated.". Otherwise the message looks like this:
> x_offset keyword deprecated.See the new offset argument.

Notice the absence of a space between the two sentences.

Same for the `GenerateContextsFile` function below.